t/t-prune.sh

```diff
@@ -1041,3 +1041,59 @@ begin_test "prune does not invoke external diff programs"
   git lfs prune
 )
 end_test
+
+begin_test "prune doesn't hang on long lines in diff"
+(
+    set -e
+
+    reponame="prune_longlines"
+    git init "$reponame"
+    cd "$reponame"
+
+    git lfs track "*.dat" 2>&1 | tee track.log
+    grep "Tracking \"\*.dat\"" track.log
+
+    echo "test data" >test.dat
+    git add .gitattributes test.dat
+
+    git commit -m tracked
+
+    git lfs untrack "*.dat" 2>&1 | tee untrack.log
+    grep "Untracking \"\*.dat\"" untrack.log
+
+    # Exceed the buffer size used by lfs.logScanner
```

Comment:
The only last thing I can think of is that I guess we should now change this comment (and the identical one in the second test below) to something like `Exceed the default buffer size that would be used by bufio.Scanner`.  Sigh.